src/views/open-api-docs-view-v2/components/version-alert/index.tsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { IconInfo16 } from '@hashicorp/flight-icons/svg-react/info-16'
+import InlineLink from 'components/inline-link'
+import PageAlert from 'components/page-alert'
+import type { OpenApiV2VersionAlertProps } from './types'
+import s from './version-alert.module.css'
+
+/**
+ * Display a version alert for API documentation
+ */
+export function OpenApiV2VersionAlert({
```

Comment:
Copied in from the previous view, this is a wrapper around our global `PageAlert`, with some logic for conditional rendering that'd otherwise have to be repeated in at least two places.

In theory we might be able to lift this up as a global component, since nothing about it is particularly specific to OpenApiDocs.